core/server/src/main/java/alluxio/master/file/FileSystemMasterClientRestServiceHandler.java

```diff
@@ -70,6 +70,9 @@
 
   private final FileSystemMaster mFileSystemMaster = AlluxioMaster.get().getFileSystemMaster();
 
+  // prevent instantiation
+  private FileSystemMasterClientRestServiceHandler() {}
```

Comment:
this needs to have a public constructor ... otherwise the jersey framework is unable to instantiate this class when creating REST endpoint handlers